src/ol/Object.js

```diff
@@ -87,7 +87,7 @@ class BaseObject extends Observable {
   constructor(opt_values) {
     super();
 
-    // Call {@link module:ol/util~getUid} to ensure that the order of objects' ids is
+    // Call {@link module:ol/util~.etUid} to ensure that the order of objects' ids is
```

Comment:
```suggestion
    // Call {@link module:ol/util.getUid} to ensure that the order of objects' ids is
```

Reply:
Thanks for the catch.  I saw the same and pushed 80181c0dcea846c7ff3472a55a2972e8cf01b076 to correct it.